### PR TITLE
urlwatch: update to 2.26

### PR DESCRIPTION
**Pull request by RunningDroid**
#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

File: srcpkgs/urlwatch/template [hunk 1]
```diff
@@ -14,8 +14,9 @@ short_desc="Tool for monitoring webpages for updates"
 maintainer="RunningDroid <runningdroid@zoho.com>"
 license="BSD-3-Clause"
 homepage="https://thp.io/2008/urlwatch/"
+changelog="https://raw.githubusercontent.com/thp/urlwatch/master/CHANGELOG.md"
 distfiles="${PYPI_SITE}/u/urlwatch/urlwatch-${version}.tar.gz"
-checksum=6802297d3318286e7f3d36b9a4567a2fb09b0ae779d4b76811dd29a7281c1f8a
+checksum=b14bbe23d87b7d4ac8b4e71a51fd2066908a947ccf7b557c3e4f87b09f455e8f
```
Comment (mhmdanas) [inline srcpkgs/urlwatch/template:20]: Remove `do_check` below and do this:
```suggestion
# skip the tests that require python modules that aren't packaged
# (pdftotext & pytesseract)
make_check_args="-k not((pdf)or(ocr))"

```

**Review by mhmdanas — CHANGES_REQUESTED** [review]
The checks worked for me locally without `python3-Pygments` in checkdepends, so remove it from there.

**Comment by RunningDroid** [comment]
I stumbled across a [regression](https://github.com/thp/urlwatch/issues/755) when I tried to add upstream's dependency docs to my local urlwatch*, so I'd prefer to wait and see what upstream does before merging but I'm not opposed to merging as-is.

*: I have an [alias set up](https://gitlab.com/RunningDroid/minidots/-/blob/master/.zsh/aliases#L29-31) for some sites, I only came across this because I tried to add it to urlwatch without an alias.

**Comment by RunningDroid** [comment]
I've pulled in upstream's patch for the issue I came across, this is ready to merge whenever anyone has time

**Pull request merged**